data/config/param_item_modifiers.yaml

```diff
@@ -44,3 +44,8 @@ META:
       required: false
       description: One or more values to be stored for this Meta Name
       values: .*
+OVERLAP:
+  summary: This item is allowed to overlap the previous item
+  description: If an item's bit offset overlaps another item, COSMOS issues a warning. This keyword explicitly
+    allows an item to overlap another and supresses the warning message.
+  since: 4.4.1
```

Comment:
Guessing here. Maybe 4.5.0? If you do the release you'll have to update this.

Reply:
@ryanatball what do you think the next release version will be?

Reply:
I don't have any incompatible changes planned so 4.4.1 is still accurate.